Vic2ToHoI4/Source/V2World/CultureGroups.cpp

```diff
@@ -66,11 +43,11 @@ void Vic2::cultureGroups::init(const Configuration& theConfiguration)
 		}
 	});
 
-	LOG(LogLevel::Info) << "Determining culture groups";
+	Log(LogLevel::Info) << "Determining culture groups";
 
 	for (auto mod: theConfiguration.getVic2Mods())
 	{
-		LOG(LogLevel::Debug) << "Reading mod cultures from " << mod;
+		Log(LogLevel::Info) << "\tReading mod cultures from " << mod;
```

Comment:
wait what? No checking if file exists down below? That'll crash for any mod that doesn't have cultures.